### ISSUE TITLE
Tensorflow Object detection -> How to get class name from trained model (.pb file)

### ISSUE THREAD

**Issue by dhgokul** [tensorflow/tensorflow#23717]
Guys,

Tensorflow Object detection -> How to get class name from trained model (.pb file)
Trained with four class, i want to know how to get class name based on input give.
I checked with existing trained model and can able to get index value , but not able to get index/class name for my own trained mode

Any help appreciated !

**Comment by wt-huang** [comment]
You can invoke model.predict() if using Keras, then call `probas.argmax(axis=-1)`

A similar issue [2291](https://github.com/tensorflow/models/issues/2291) may help your case.


Here is the issue template to fill out:

**System information**

Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
TensorFlow installed from (source or binary):
TensorFlow version (use command below):
Python version:
Bazel version (if compiling from source):
GCC/Compiler version (if compiling from source):
CUDA/cuDNN version:
GPU model and memory:
Exact command to reproduce:
You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with:

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"
Describe the problem

Describe the problem clearly here. Be sure to convey here why it's a bug in TensorFlow or a feature request.

Source code / logs

Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached. Try to provide a reproducible test case that is the bare minimum necessary to generate the problem.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @wt-huang: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.
